tests/test_serverless/test_modules/test_ping.py

```diff
@@ -67,5 +67,5 @@ async def test_start_ping(self, mock_get_return):
         mock_get_return.side_effect = aiohttp.ClientError("Test Error")
 
         with patch("runpod.serverless.modules.rp_ping.log.error") as mock_log_error:
-            await new_ping._send_ping() # pylint: disable=protected-access
-            assert mock_log_error.call_count == 2
+            await new_ping.start_ping(test=True)
+            assert mock_log_error.call_count == 1
```

Comment:
Great improvement. Tests should definitely use the exposed methods, and not methods meant to be private/protected.